draft-ietf-core-problem-details.md

```diff
@@ -30,9 +30,6 @@ author:
     email: cabo@tzi.org
 
 normative:
```

Comment:
Instead of removing STD94, we may want to use it at the right place.

Reply:
Done in [164b1d4](https://github.com/core-wg/core-problem-details/pull/16/commits/164b1d4c930eb3088cdaa257206bd207158cac4d)